apps/game_backend/lib/game_backend/matches.ex

```diff
@@ -114,6 +100,36 @@ defmodule GameBackend.Matches do
     end)
   end
 
+  defp process_bounties(multi, results) do
```

Comment:
Please avoid this generic naming. We could use something like `give bounties rewards`, it follows the previous piped function (give_rewards) syntax.
or, something else.

Reply:
Sure thing, will about things like this in the future